club-website/src/app/about/page.tsx

```diff
@@ -1,4 +1,4 @@
-import { poppins } from "../fonts";
+git import { poppins } from "../fonts";
```

Comment:
remove "git " from here ^_^ not sure how that got there lol

Reply:
fixed!